datastore/redis.go

```diff
@@ -216,8 +216,17 @@ func (r *RedisCache) GetKnownValidators() (map[uint64]boostTypes.PubkeyHex, erro
 	return validators, nil
 }
 
+func (r *RedisCache) SetMultiKnownValidator(indexPkMap map[uint64]boostTypes.PubkeyHex) error {
+	values := []string{}
+	for proposerIndex, publickeyHex := range indexPkMap {
+		values = append(values, strconv.FormatUint(proposerIndex, 10), PubkeyHexToLowerStr(publickeyHex))
```

Comment:
why convert proposerIndex to string here?

Reply:
"values" is a string slice.
Considering it will eventually be converted to a string (no Int in Redis) this is the cleanest approach.